codec-http/src/test/java/io/netty/handler/codec/http/websocketx/WebSocketHandshakeHandOverTest.java

```diff
@@ -95,6 +118,64 @@ protected void channelRead0(ChannelHandlerContext ctx, Object msg) throws Except
         assertTrue(clientReceivedMessage);
     }
 
+    @Test
```

Comment:
Use a timeout as we may never complete the loop here when there is a bug:

https://github.com/netty/netty/pull/8896/files#diff-96a945171dadb844c094502b4d37149bR164